tests/BuildScriptGenerator.Tests/DotnetCore/ProjectFileProviderTestBase.cs

```diff
@@ -37,6 +37,27 @@ public abstract class ProjectFileProviderTestBase : IClassFixture<TestTempDirTes
         </Project>
         ";
 
+        protected const string AzureBlazorWasmClientProjectFile = @"
+        <Project Sdk=""Microsoft.NET.Sdk.Web"">
+          <PropertyGroup>
+            <TargetFramework>netstandard2.1</TargetFramework>
+            <RazorLangVersion>3.0</RazorLangVersion>
+          </PropertyGroup>
+          <ItemGroup>
+            <PackageReference Include=""Microsoft.AspNetCore.Components.WebAssembly"" Version=""3.2.0-rc1.20223.4"" />
```

Comment:
Could you add one test case where this package reference is not there and only netstandard2.1 is present..should not be detected as wasm project

Reply:
ok